browser/ui/views/frame/brave_browser_view.h

```diff
@@ -82,6 +83,7 @@ class BraveBrowserView : public BrowserView {
   vertical_tab_strip_widget_delegate_view() {
     return vertical_tab_strip_widget_delegate_view_;
   }
+  void ShowBraveHelpBubbleView(const std::u16string text);
```

Comment:
We're not modifying the text arg inside the function isn't it best to pass by value here?

Reply:
Oh, yeah I think so. We're trying to avoid copying objects. we prefers passing objects arguments in `const &` if they're not primitive types or a few exceptions like base::Time().

Reply:
As per a comment, base::Time occupies 64bits so passing by reference or pointer don't have benefit. https://source.chromium.org/chromium/chromium/src/+/3abcabe0cd77a5a28abbcbd9ba788a63cf9af36c:base/time/time.h;l=25